tests/net/ieee802154/l2/src/ieee802154_test.c

```diff
@@ -535,6 +535,66 @@ static bool test_packet_cloning_with_cb(void)
 	return true;
 }
 
+static bool test_packet_rssi_conversion(void)
+{
+	struct net_pkt *pkt;
+	uint8_t raw_rssi;
+	int8_t raw_signed_rssi;
+
+	NET_INFO("- RSSI conversion between unsigned and signed representation\n");
+
+	pkt = net_pkt_rx_alloc_on_iface(iface, K_NO_WAIT);
+	if (!pkt) {
+		NET_ERR("*** No pkt to allocate\n");
+		return false;
+	}
+
+	/* Test setting/getting of unsigned RSSI. */
+	net_pkt_set_ieee802154_rssi(pkt, 50U);
+	zassert_equal(net_pkt_ieee802154_rssi(pkt), 50U);
+
+	/* Test setting/getting of signed RSSI (in range). */
+	net_pkt_set_ieee802154_rssi_dbm(pkt, -174);
```

Comment:
Looks great - thanks for adding to the testsuite :-)

The only suggestion I have is to use `#define`ed constant suggested earlier.

Reply:
Fixed.